catalog/java/io/material/catalog/slider/SliderMainDemoFragment.java

```diff
@@ -37,6 +39,19 @@ public View onCreateDemoView(
         layoutInflater.inflate(R.layout.cat_slider_fragment, viewGroup, false /* attachToRoot */);
 
     Slider slider = view.findViewById(R.id.slider);
+    slider.setOnSliderTouchListener(new Slider.OnSliderTouchListener() {
```

Comment:
I think just adding this behavior to the main demo only is enough. Could you remove it from the other demos?

Reply:
Sure, I can do that :)

Reply:
Done